### DISCUSSION TITLE
lesson 6: from which github repo do i get the getEntranceFee function

### DISCUSSION THREAD

**Discussion by rishigarg256** [smartcontractkit/full-blockchain-solidity-course-py#1398] [Q&A] [answered]
I am having troubles understanding what is this getEntranceFee function and from which github repo do i get it from

**Comment by cromewar** [comment, marked as answer]
Hello @rishigarg256 
The get Entrance fee of lesson 6 is just a bunch of math to set the entrance fee to $50 USD dollars based on the price given by `getPrice` but at the end is just a bunch of math.
